cypress/e2e/a11y.js

```diff
@@ -1,10 +1,10 @@
 import { reportA11y } from '../support/accessibility'
 
 describe('a11y', () => {
-    it('home should have no accessibility violations', () => {
+    it.only('home should have no accessibility violations', () => {
```

Comment:
If _only_ this wasn't here it would be the perfect PR

Reply:
20 🦔 pun points :D